<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RollWidget</class>
 <widget class="QWidget" name="RollWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RollWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QListWidget" name="resultView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="rollEdit">
       <property name="placeholderText">
        <string>Example: 2d20 + 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commandButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Roll!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="compactModeBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Compact mode</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="d4Button">
       <property name="text">
        <string>d4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d6Button">
       <property name="text">
        <string>d6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d8Button">
       <property name="text">
        <string>d8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d10Button">
       <property name="text">
        <string>d10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d12Button">
       <property name="text">
        <string>d12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d20Button">
       <property name="text">
        <string>d20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d100Button">
       <property name="text">
        <string>d100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
